model/metrics/k8s.yaml

```
@@ -0,0 +1,40 @@
+groups:
+  # k8s.pod.cpu.* metrics and attribute group
```

nit:

```suggestion
  # k8s.pod.cpu.* metrics
```

(seems there are no attributes)